public/js/updateStudentProfile.js

```diff
@@ -0,0 +1,23 @@
+const edaidEditButton = document.querySelector('.edaidEditButton')
+const upfrontEditButton = document.querySelector('.upfrontEditButton')
+const laptopEditButton = document.querySelector('.laptopEditButton')
+const githubUserEditButton = document.querySelector('.githubUserEditButton')
+
+function handleEditClick(event) {
+    const buttonName = event.target.getAttribute('class')
+    const divName = buttonName.replace('EditButton', 'Container')
+    const divSelector = document.querySelector('.' + divName)
```

Comment:
it's not a div selector, it's just the div. so e.g. containerDiv

Reply:
done👍